docs/src/main/tut/index.md

```diff
@@ -98,7 +98,7 @@ Many projects integrate with cats. By sharing the same set of
 type classes, instances and data types, projects can speak the same "cats
 language", and integrate with each other with ease. 
 
-#### Generic libraries to support pure functional programming
+#### General purpose libraries to support pure functional programming
 
  * [Mouse](https://github.com/benhutchison/mouse): a small companion to cats that provides convenient syntax (aka extension methods)
```

Comment:
Mouse can be removed from this list, as it's listed above as one of the cats extra modules. Also, it's not on this list in the primary README anyway.